jwst/master_background/master_background_step.py

```diff
@@ -140,6 +146,12 @@ def process(self, input):
                         exptime_key='exposure_time',
                     )
 
+                    # If requested, apply a moving-median boxcar filter to the master background spectrum
+                    if (self.median_kernel > 1):
+                        self.log.info('Applying moving-median boxcar of width %i',self.median_kernel)
+                        master_background.spec[0].spec_table['surf_bright'] = medfilt(
+                            master_background.spec[0].spec_table['surf_bright'], kernel_size=[self.median_kernel])
```

Comment:
I understand not doing a median filter on the error/variance arrays since that operation is poorly defined, but should we also median filter the flux array just for consistency in case user's save the 1d background spectrum?

Reply:
Hm, is the FLUX array being written out as an option?  I think only the SURF_BRIGHT extension is relevant here as the FLUX array shouldn't be getting used in the background step?